modules/ROOT/pages/api/overview.adoc

```diff
@@ -57,4 +57,35 @@ An `X-Request-Id` response header is returned with each request and can be used
 
 The value of this header contains a unique ID that can be used to track the journey of a request.
 
-If you run into any issues with a particular request, you can https://support.neo4j.com/[raise a support ticket] and provide the `X-Request-Id`.
\ No newline at end of file
+If you run into any issues with a particular request, you can https://support.neo4j.com/[raise a support ticket] and provide the `X-Request-Id`.
+
+== Query API
```

Comment:
Not sure if the Query API belongs in the Aura API. We could maybe change the top level header to _API_ and then have separate sections on Aura API and Query API.